CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 VAST now merges the contents of all used configuration files instead of
+  using only the most user-specific file.
```

Comment:
Can you please elaborate about the merge order explicitly? It's only intuitively clear, but I think we should be precise how this happens.